registry/manifests/kabanero-operator/0.4.0/kabanero-operator.v0.4.0.clusterserviceversion.yaml

```diff
@@ -52,7 +52,7 @@ spec:
   apiservicedefinitions: {}
   maturity: alpha
   version: 0.4.0
-  replaces: kabanero-operator.v0.3.1
+  replaces: kabanero-operator.v0.3.5
```

Comment:
This is not going to hurt anything, but it's also not necessary, because we also specify a skiprange on the next minor version.